src/app/shared/reactions/reactions.component.ts

```diff
@@ -19,20 +20,35 @@ export class ReactionsComponent implements OnChanges {
   @Input() showAmounts: boolean = true;
   @Input() showBar: boolean = true;
   @Input() smallIcons: boolean = false;
+  @Input() reactions: { [id: string]: number };
+  @Input() reaction: string;
```

Comment:
This name is confusing.
Better change to userReaction.
Type can be changed as well to ReactionType enum

Reply:
Done